src/style.css

```
@@ -0,0 +1,97 @@
+body {
+  margin: 0 auto;
+}
+
+.card {
+  display: flex;
+}
+
+.card-item {
+  font-family: "Roboto";
```

Brackets are necessary.
Add additional styles.
```suggestion
  font-family: Roboto, Arial, Helvetica, sans-serif;
```